pkg/cluster/k8sres.go

```diff
@@ -1523,7 +1592,8 @@ func (c *Cluster) generateLogicalBackupJob() (*batchv1beta1.CronJob, error) {
 		false,
 		"",
 		c.OpConfig.AdditionalSecretMount,
-		c.OpConfig.AdditionalSecretMountPath); err != nil {
+		c.OpConfig.AdditionalSecretMountPath,
+		[]acidv1.AdditionalVolume{}); err != nil {
```

Comment:
This should also bring in the same additional volumes here.

Reply:
Interesting question. I believe it makes sense only if we're going to provide a separate set of "additional" volumes here, probably it's not a good idea to mix volumes for Postgres and for logical backup together. I would suggest we will finish with this PR first, and the open a follow up about doing so for backup, what do you think about this plan?

Reply:
That would be a departure from the current path considering that additonalSecretMount/Path is mounted on both pods. I don't see any reason to not have these the same.